config.py

```diff
@@ -12,14 +12,40 @@
 
 TESTING = os.environ.get("TESTING")
 DEBUG = os.environ.get("DEBUG")
+SERVER_TYPE = os.environ.get("SERVER_TYPE")
 
 # Database configuration
+DB_NAME = (
+    os.environ.get("DB_NAME")
+    if SERVER_TYPE != "test"
+    else os.environ.get("TEST_DB_NAME")
+)
+DB_USER = (
+    os.environ.get("DB_USER")
+    if SERVER_TYPE != "test"
+    else os.environ.get("TEST_DB_USER")
+)
+DB_HOST = (
+    os.environ.get("DB_HOST")
+    if SERVER_TYPE != "test"
+    else os.environ.get("TEST_DB_HOST")
+)
+DB_PWD = (
+    os.environ.get("DB_PASSWORD")
+    if SERVER_TYPE != "test"
+    else os.environ.get("TEST_DB_PASSWORD")
+)
+DB_PORT = (
+    os.environ.get("DB_PORT")
+    if SERVER_TYPE != "test"
+    else os.environ.get("TEST_DB_PORT")
+)
```

Comment:
Hey @thebelrobot I see a possibility for minimizing code duplicacy a bit, can we reduce it here?

`if SERVER_TYPE != "test"` - this condition can be utilized once for all tha Environment variable, unless, we will endup duplicating this conditioning the future addition of any env variables.